src/test/java/com/ibanity/apis/client/helpers/IbanityClientSecuritySignaturePropertiesKeys.java

```diff
@@ -0,0 +1,8 @@
+package com.ibanity.apis.client.helpers;
+
+public interface IbanityClientSecuritySignaturePropertiesKeys {
+    String IBANITY_CLIENT_SIGNATURE_CERTIFICATE_PATH_PROPERTY_KEY = "client.signature.certificate.path";
+    String IBANITY_CLIENT_SIGNATURE_CERTIFICATE_ID_PROPERTY_KEY = "client.signature.certificate.id";
+    String IBANITY_CLIENT_SIGNATURE_PRIVATE_KEY_PATH_PROPERTY_KEY = "client.signature.private-key.path";
+    String IBANITY_CLIENT_SIGNATURE_PRIVATE_KEY_PASSPHRASE_PROPERTY_KEY = "client.signature.private-key.passphrase";
```

Comment:
we need to change "PASSPHRASE", eg "PPHRASE" because this gives a security score "E" on sonarcloud...

Reply:
Moved to an interface.